src/Components/Admins/Form/form.module.css

```diff
@@ -1,26 +1,17 @@
-.container {
+.formContainer {
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  position: fixed;
-  z-index: 1;
-  left: 0;
-  top: 0;
-  width: 100vw;
-  height: 100vh;
-  overflow: auto;
-  background-color: #000000b3;
-}
-
-.formContainer {
-  width: 50%;
+  padding: 40px 0px;
+  width: 100%;
+  margin: 0 auto;
 }
 
 .header {
-  display: flex;
-  justify-content: space-between;
+  width: 25vw;
   align-items: center;
+  text-align: center;
   padding: 0.5rem;
```

Comment:
Based on good practices and conventions, the content related to box model should come before the content related to text-rules. So padding should be before text-align